prov/sockets/src/sock_fabric.c

```diff
@@ -634,6 +622,62 @@ int sock_node_getinfo(const char *node, const char *service, uint64_t flags,
 	return ret;
 }
 
+static int sock_match_src_addr(struct slist_entry *entry, const void *src_addr)
```

Comment:
Er... you may not be able to - I see that the function is passed as a parameter below